inst/help/exploratoryfactoranalysis.md

```diff
@@ -10,12 +10,11 @@ With Exploratory Factor Analysis it is possible to identify one or more factors
 - There is no multicollinearity and singularity in the data. 
 
 ### Input 
----
```

Comment:
why did you delete this?

Reply:
in my viewer it did not render this properly, but I will put it back.

Reply:
not in JASP, or in some other viewer? They show up fine for me.

Reply:
Inside R it displayed weirdly

Reply:
Then please put it back, as JASP has its own viewer for markdown/ HTML.